File: Php/Car.php
<?php
class Car {
    public $id = integer;
    public $license = string;
    public $driver = string;
    public $passegenger = integer;

    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }

    public function PrintDataCar(){
        echo "license: $this->license, conductor: {$this->driver->name}, document:{$this->driver->document}";
    }
}

?>

File: Java/Car.java
package Java;

class Car {
    Integer id;
    String license;
    Account driver;
    private Integer passegenger;

    public Car (String lincense, Account driver) {
        this.license = license;
        this.driver = driver;
        passegenger = 4; 
        System.out.print("passenger: " + passegenger);
    }

    void printDataCar() {
        System.out.println("Lincense: " + license + "Driver: " + driver.name + " Passengers: " +  passegenger);
    }
}
